### DISCUSSION TITLE
how to disable spellcheck in markdown file?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#4021] [Q&A] [answered]
I already add 
```lua
vim.api.nvim_create_autocmd("FileType", {
  pattern = "markdown",
  callback = function()
    vim.opt_local.spell = false
  end
})
```
to `init.lua`

And `vim.opt_local.spell=false`
to `config/nvim/lua/ftplugin` 

But neither work for me.

**Comment by dpetka2001** [comment, marked as answer]
You have to put the autocmd in `~/.config/nvim/lua/config/autocmds.lua`. Kindly read the docs of LazyVim website.

**Reply by FLeaooo** [reply]
For some reason is working for makdown but not to files txt

```
vim.api.nvim_create_autocmd("FileType", {
  pattern = { "markdown", "txt" },
  callback = function()
    vim.opt_local.spell = false
  end,
})
```

**Reply by OriLiMu** [reply]
> For some reason is working for makdown but not to files txt
> 
> ```
> vim.api.nvim_create_autocmd("FileType", {
>   pattern = { "markdown", "txt" },
>   callback = function()
>     vim.opt_local.spell = false
>   end,
> })
> ```

the filetype is text

**Comment by Alilacream** [comment]
thank you for the clear explanaition guys 😄
